tests/integration/curl/CurlTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Copyright 2018 OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenCensus\Tests\Integration\Trace;
+
+use OpenCensus\Trace\Tracer;
+use OpenCensus\Trace\Exporter\ExporterInterface;
+use OpenCensus\Trace\Integrations\Curl;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @group trace
+ */
+class CurlTest extends TestCase
+{
+    public static function setUpBeforeClass()
+    {
+        Curl::load();
+    }
+
+    public function setUp()
+    {
+        if (!extension_loaded('opencensus')) {
+            $this->markTestSkipped('Please enable the opencensus extension.');
+        }
+        opencensus_trace_clear();
+    }
+
+    public function testCurlExec()
+    {
+        $url = 'https://www.google.com/';
```

Comment:
I think in here we could avoid this Google dependency. Would you consider to use something like https://github.com/jcchavezs/httptest-php

Reply:
Yes, I think that would work.

Reply:
We'll skip this for now: https://github.com/jcchavezs/httptest-php/issues/5

Reply:
@chingor13 I just opened a PR addressing this issue: https://github.com/jcchavezs/httptest-php/pull/6